packages/@expo/cli/src/start/server/metro/createServerRouteMiddleware.ts

```diff
@@ -110,6 +112,11 @@ export function createRouteHandlerMiddleware(
         logMetroError(projectRoot, { error });
       },
       async getApiRoute(route) {
+        const { exp } = getConfig(projectRoot, { skipSDKVersionRequirement: true });
```

Comment:
This method can take multiple seconds to execute in the ncl app, we should only load it once. Pass in the one loaded in MetroBundlerDevServer.